src/Diagram/index.js

```diff
@@ -160,6 +160,7 @@ export class DiagramSection extends Component {
 
 export default DiagramSection
 
+/* eslint-disable max-len */
 const Diagram = styled.section`
   padding-top: 80px;
```

Comment:
I also don't agree this is necessary. Is there a problem with how ESLint formats this? I see the `@media...` line 210 below is already in a single line anyway.

Reply:
...which BTW goes over 80 chars, which is the problem `eslint` reported for this file. See https://github.com/iterative/dvc.org/issues/548#issuecomment-544209275

Reply:
Same for the other 3 files where this is done.

Reply:
@jorgeorpinel So should I let it stay above 80 characters and remove the disabling comments or shall I stick with disabling linting on media queries?

Reply:
I don't understand. If you disable it, then the linter won't break the line. Simply do not disable the linter and let it break the line?

Reply:
@jorgeorpinel linter wasn't breaking the line and was throwing an error. But when I manually fixed it, it reverted the changes forcefully and threw a `max-len` error.

Reply:
The linter only examines Git changes, not all the files every time. You would have to run it manually like `yarn prettier --write '{static,src}/**/*.{js,jsx,md}`. (Mentioned in https://dvc.org/doc/user-guide/contributing-docs#doc-style-guidelines-and-tips-for-java-script-and-markdown)

Reply:
So can you just break those lines manually and remove all these `/* eslint-disable max-len */`? 🙂

Reply:
@jorgeorpinel I did break those lines. But when I ran the linter, it reverted my changes and put everything in one line.
I broke the media queries and the tests function.

Reply:
Ohhhh! This is not CSS, it's a template literal in JavaScript... But still I didn't know Prettier would elongate some lines. That's not nice! Seems like it's a known problem though... Check prettier/prettier/issues/3368, prettier/prettier/issues/6192, and prettier/prettier/issues/6262 for possible workarounds.

Try https://github.com/sgtpep/prettier-plugin-html-template-literals...

Not sure what to do otherwise, I guess just let those line be long. We definitely don't want `eslint-disable max-len`.

Reply:
@jorgeorpinel 
https://github.com/prettier/prettier/pull/5979
I think this PR is forcing prettier to make a template literal into one line :man_facepalming: 
Check out this [changelog](https://github.com/prettier/prettier/pull/5979/files#diff-d310d8d326c0b4526a13ac21c13110a2).
[This guy's](https://github.com/prettier/prettier/issues/6192#issuecomment-525723143) current workaround is using `// prettier-ignore`

Reply:
OK just remove the eslint ignore comments for now please, and merge master so this PR is mergeable (except for the CI lint test which will not pass).